Source/WebKit/NetworkProcess/cocoa/NetworkSessionCocoa.h

```diff
@@ -168,6 +168,8 @@ class NetworkSessionCocoa final : public NetworkSession {
     void applyProxyConfigurationToSessionConfiguration(NSURLSessionConfiguration *);
 #endif
 
+    bool isNetworkSessionCocoa() const final { return true; }
```

Comment:
Should be private, makes no sense for someone to call this one the subclass (only on the base class).